### DISCUSSION TITLE
Set custom URL path/string

### DISCUSSION THREAD

**Discussion by SpuriousGer** [pglombardo/PasswordPusher#2425] [Ideas]
It would be cool to be able to set the /p/<URLstring> to something individual. I hope I have not overlooked the feature, but it seems like something that would be worthwhile for ease of use sharing.

**Comment by pglombardo** [comment]
Hi @SpuriousGer - thanks for the suggestion?  What would be this be useful for?  For you or end users?  Just want to make sure I understand the reasoning.

**Comment by SpuriousGer** [comment]
Hi @pglombardo,

two use cases for me (I am actively using this feature on a similar website, but love that this is self-hosted):

1. My work computer is locked down and I can't easily send text back and forth. Having a very simple URL with a one time usage attached, would make this so easy.
2. I am frequently working with freelancers on fiverr and having the same link to share stuff would be very convenient to share stuff. Fiverr intercepts the URLs, it takes ages, they know the cotent which you sometimes don't want etc. Telling someone "check your URL" is very convenient.

**Reply by pglombardo** [reply]
I understand - this is a great idea.  I'll see if I can get to this sooner than later.  I'll update here once I have something.
